semgrep/semgrep/util.py

```diff
@@ -198,5 +198,5 @@ def truncate(file_name: str, col_lim: int) -> str:
     name_len = len(file_name)
     prefix = "..."
     if name_len > col_lim:
-        file_name = prefix + file_name[name_len - col_lim + len(prefix) :]
+        file_name = prefix + file_name[name_len - col_lim + len(prefix):]
```

Comment:
```suggestion
        file_name = prefix + file_name[name_len - col_lim + len(prefix) :]
```
Remove the lint fix here